gatsby-node.js

```diff
@@ -7,9 +7,13 @@ exports.onCreateBabelConfig = ({ actions }, pluginOptions) => {
 };
 
 exports.onCreateWebpackConfig = ({ actions, stage }) => {
-  if (stage === "develop" || stage === "build-javascript") {
+  if (
+    stage === "develop" ||
+    stage === "build-javascript" ||
+    stage === "build-html"
```

Comment:
this needs `develop-html` too as well for dev ssr

Reply:
Yes. That would be all stages this callback runs. I removed this condition as it can cause bugs later, if more stages are added. 
For some reason it worked with DEV_SSR enabled.